### PR TITLE
fix : add support arm64 (darwin, linux)

### PR DESCRIPTION
**Pull request by fly33499**
Hi. I've patched installer to support arm64 (darwin, linux).
Please check it and update it.

you can see the issues below.
https://github.com/ignite-hq/installer/issues/2

### CODE REVIEW

**Comment by ilgooz** [comment]
@helder-moreira can you please review?

**Review by helder-moreira — CHANGES_REQUESTED** [review]
Looks good to me in general. Thanks 🙏  

But why do we have an additional `statik` package now? Isn't it duplicated?

**Comment by fly33499** [comment]
> Looks good to me in general. Thanks 🙏
> 
> But why do we have an additional `statik` package now? Isn't it duplicated?

Yes, you're right. Now I've removed unused file. Can you check it again plz?

File: scripts/statik.go [hunk 1]
```diff
@@ -1,14 +1,14 @@
 // Code generated by statik. DO NOT EDIT.
 
-package scripts
+package statik
```
Comment (helder-moreira) [inline scripts/statik.go]: The package name should be kept as `scripts`. Can you please change this?

**Review by helder-moreira — CHANGES_REQUESTED** [review]
Just one additional comment.

File: scripts/statik.go (hunk 1 above)
Comment (fly33499) [reply scripts/statik.go]: yep!

**Comment by ilgooz** [comment]
Thank you @fly33499 for fixing this!

**Review by helder-moreira — APPROVED** [review]
LGTM

**Pull request merged**